src/main/resources/assets/tconstruct/models/block/fluid_cannon_up.json

```diff
@@ -0,0 +1,67 @@
+{
+  "render_type": "minecraft:cutout",
```

Comment:
You know, I'm pretty sure this and the other fluid cannons uses the same model as the melter, save different textures. Recent Mantle reworks means the melter no longer lists items inside so its just a generic tank.

We can probably make a template for all three cannon directions and the melter to inherit from.